Diary/Controller/DetailViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DetailViewController.swift
+//  Diary
+//
+//  Created by mmim, grumpy, mino on 2022/06/14.
+//
+
+import UIKit
+
+final class DetailViewController: UIViewController {
+    private lazy var detailView = DetailView(frame: view.bounds)
+    private let diary: Diary
+    
+    init(diary: Diary) {
+        self.diary = diary
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        super.loadView()
+        view = detailView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpNavigationBar()
+        detailView.setUpContents(data: diary)
+        detailView.scrollTextViewToTop()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillShow),
+            name: UIResponder.keyboardWillShowNotification,
+            object: nil
+        )
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillHide),
+            name: UIResponder.keyboardWillHideNotification,
+            object: nil
+        )
+    }
+    
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
```

Comment:
이렇게 Notification을 등록했다가 제거해주는 이유는 뭔가요?

`If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function. If you forget or are unable to remove an observer, the system cleans up the next time it would have posted to it.`

라고 공식문서에 적혀있네요.
https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver

Reply:
오 감사합니다 :) 옵저버 제거하는 코드 삭제했습니다.